bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/ArithmeticGroupFunction.java

```diff
@@ -573,6 +580,20 @@ public State getStateAs(Set<Item> items, Class<? extends State> stateClass) {
         public State[] getParameters() {
             return new State[] { upperState, lowerState, upperLimit, lowerLimit };
         }
+        
+		private String getFunctype(Item item) {
+			String memberFuncType = "Undef";
+			for (String tag : item.getTags()) {
+				JsonElement jsonElementTag = jp.parse(tag);
+				if (jsonElementTag instanceof JsonObject) {
+					JsonObject jsonObjectMemberTag = ((JsonObject) jsonElementTag);
+					if (jsonObjectMemberTag.has("functype")) {
+						memberFuncType = jsonObjectMemberTag.get("functype").toString();

```

Comment:
Isn’t it simpler to use `….getAsString()` and not have to remove the surrounding `"`?